platform/InternalIOErr.roc

```diff
@@ -31,8 +30,23 @@ IOErr : [
     Other Str,
 ]
 
-handleErr : Host.InternalIOErr -> IOErr
-handleErr = \{ tag, msg } ->
+IOErrFromHost : {
+    tag : [
+        EndOfFile,
+        NotFound,
+        PermissionDenied,
+        BrokenPipe,
+        AlreadyExists,
+        Interrupted,
+        Unsupported,
+        OutOfMemory,
+        Other,
+    ],
+    msg : Str,
+}
+
+handle_err : IOErrFromHost -> IOErr
+handle_err = \{ tag, msg } ->
```

Comment:
I know this was already there, but this maps everything one-to-one _except_ for `EndOfFile`. Since this method is generically called `handle_err`, this seems like a bad idea, but maybe there's a good reason.